doctest/parts/doctest_impl.h

```diff
@@ -1430,7 +1430,7 @@ namespace {
             sigStack.ss_size  = sizeof(altStackMem);
             sigStack.ss_flags = 0;
             sigaltstack(&sigStack, &oldSigStack);
-            struct sigaction sa = {nullptr};
+            struct sigaction sa = {0};
```

Comment:
Shouldn't it be `struct sigaction sa{}`?

I'm not sure if this initializes all the struct members, whereas value-initialization will.